### PR TITLE
Ali/ci-unittests

### PR DESCRIPTION
**Pull request by m-ali-0001**

### CODE REVIEW

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
skip ci

File: cms/envs/test.py [hunk 1]
```diff
@@ -122,10 +122,19 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': "edxtest",
+        'NAME': 'edxapp',
```
Comment (emraan004) [inline cms/envs/test.py:125]: Will it use edx-platform database instead of test database?

File: lms/envs/test.py [hunk 2]
```diff
@@ -176,19 +176,19 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.mysql',
-        'NAME': 'edxtest',
+        'NAME': 'edxapp',
```
Comment (emraan004) [inline lms/envs/test.py:179]: Again here same above wala comment

File: lms/envs/test.py [hunk 3]
```diff
@@ -656,3 +656,32 @@
...
+
+ENV_TOKENS = dict()
+
+if ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR'):
+    COMPREHENSIVE_THEME_DIR = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIR')
+
+COMPREHENSIVE_THEME_DIRS = ENV_TOKENS.get('COMPREHENSIVE_THEME_DIRS', COMPREHENSIVE_THEME_DIRS) or []
+COMPREHENSIVE_THEME_LOCALE_PATHS = ENV_TOKENS.get('COMPREHENSIVE_THEME_LOCALE_PATHS', [])
+
+NODEBB_MASTER_TOKEN = "1dae9ee7-3b32-4d17-a35b-c179a1788ec3"
```
Comment (emraan004) [inline lms/envs/test.py:683]: Why are these tokens exposed here? why are we not accessing them through settings?

Comment (m-ali-0001) [reply lms/envs/test.py:683]: I was instructed by Qasim and Ahmed to add these here as a temporary measure. I think development team was working on reading this from environments or from some settings file.

File: cms/envs/test.py (hunk 1 above)
Comment (m-ali-0001) [reply cms/envs/test.py:125]: No. 
Everytime we run untitest a new EC2 instance is launched which has an empty "edxapp" database in it. This is entirely different from edxapp database of dev, staging or other environments.

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by m-ali-0001** [comment]
jenkins run all

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by ahmed-faraz46** [comment]
Code is merged under https://github.com/philanthropy-u/edx-platform/pull/1724.

**Pull request closed without merging**